### DISCUSSION TITLE
Welcome to ngx-jodit Discussions!

### DISCUSSION THREAD

**Discussion by julianpoemp** [julianpoemp/ngx-jodit#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by kemotx90** [comment]
Good evening, would you be collaborate with me for merge our libraries and try to create a better product for angular and Jodit usage?

**Reply by julianpoemp** [reply]
@kemotx90 good evening. Thanks for the offer, but I'll stay with this library as it is. If you see any missing feature you can create a new issue and contribute with PRs.
